src/ensembl/src/content/app/custom-download/types/Attributes.ts

```diff
@@ -1,21 +1,36 @@
+import { RadioOptions } from 'src/shared/radio/Radio';
+import { Option } from 'src/shared/select/Select';
+import { CheckboxGridOption } from 'src/content/app/custom-download/components/checkbox-grid/CheckboxGrid';
+import JSONValue from 'src/shared/types/JSON';
+
 export type Attribute = {
-  isChecked: boolean;
-  id: string;
+  type: string;
```

Comment:
Shouldn't this be an enum? Isn't there a very limited range of attribute types?

Reply:
That's done